packages/react-api/src/types.ts

```diff
@@ -54,7 +54,9 @@ export interface CallState {
 
 export type CallProps = ApiProps & CallState;
 
-export type BaseProps<T> = BareProps & CallProps & ChangeProps & {
+export type BaseProps<T> = BareProps &
+CallProps &
+ChangeProps & {
```

Comment:
```suggestion
export type BaseProps<T> = BareProps & CallProps & ChangeProps & {
```

Actually weird, thought we manager to get rid of all the type with `&` and replaced with `interface`. (Probably some old reason behind this one)